lib/ex_doc/language/erlang.ex

```diff
@@ -215,6 +215,24 @@ defmodule ExDoc.Language.Erlang do
     binary
   end
 
+  defp walk_doc({:code, attrs, [{:a, a_attrs, [a_inner], _a_meta} | _] = a, meta}, _config) do
+    {arity, extra} =
+      case :lists.keydelete(:a, 1, a) do
+        [] ->
+          {0, ""}
+
+        [extra] ->
+          # a_meta is not broken down, so we have to parse stuff like (X, Y)
+          {extra |> String.split(",") |> Enum.count(), extra}
+      end
```

Comment:
To me this seems like a silly way to calculate the arity, but I couldn't find a better one.

Reply:
You can calculate the arity but you'll have to reduce, here's an example of what a non zero arity would look like : 

```
     {code,[],
        [<<"{that_odd_form, ">>,
         {a,
          [{href,
            <<"_build/default/lib/stdlib/doc/queue.html#type-queue">>}],
          [<<"queue:queue">>]},
         <<"(">>,
         {a,
          [{href,
            <<"_build/default/lib/stdlib/doc/uri_string.html#type-uri_string">>}],
          [<<"uri_string:uri_string()">>]},
         <<")}">>]}
```         

Or with a built-in 

```
      {code,[],
        [<<"{that_odd_form, ">>,
         {a,
          [{href,
            <<"_build/default/lib/stdlib/doc/queue.html#type-queue">>}],
          [<<"queue:queue">>]},
         <<"(any())}">>]}
```

Whether it's worth it I suppose is another question :)